en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
+  * [Airframes](airframes/README.md)
+    * [Airframes Reference](airframes/airframe_reference.md)
+    * [Adding a New Airframe](airframes/adding_a_new_frame.md)
+    * [Parrot Bebop](advanced/parrot_bebop.md)
+  * [Porting Guide](debug/porting-guide.md)
+  * [Data Links](data_links/README.md)
```

Comment:
Are there other data links that Telemetry Wifi? What?

Reply:
The ones mentioned here are the main ones, but basically anything that has an UART interface or UDP (on a linux board) can be used as data link for mavlink.
For non-mavlink, there's also frsky telemetry: https://docs.px4.io/en/peripherals/frsky_telemetry.html, and we have a driver for Blacksheep Telemetry as well (I never used it).